src/js/11-elastic-search.js

```diff
@@ -132,6 +169,25 @@ function toggleSearchBar () {
   }
 }
 
+function toggleFilterContainer () {
+  const facetsSidebar = document.getElementById('facets-sidebar')
+  const mediaQuery = window.matchMedia('(min-width: 1024px)')
+
+  if (facetsSidebar.classList.contains('d-none')) {
+    facetsSidebar.classList.remove('d-none')
+    if (!mediaQuery.matches) {
+      document.body.style.position = 'fixed'
+      document.body.style.top = `-${window.scrollY}px`
+    }
+  } else {
+    facetsSidebar.classList.add('d-none')
+    if (!mediaQuery.matches) {
+      document.body.style.position = ''
+      document.body.style.top = ''
+    }
+  }
+}
```

Comment:
To toggle an element, i would just toggle a single css-class and define its behavior in your css files.